config/config_notwin_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
I think not, because unix is not a **$GOOS** valid value. BTW I got inspiration to use this name from here: _$GOROOT/src/internal/testenv_
Of course, it could be changed, but I'll need to add a list of all the unix like operating systems instead of using the **!windows**. Here is an example of your recommendation: _$GOROOT/src/os/env_*test.go_
If you still think that the approach follow by _env_*test.go_ is better, please let me know and I'll be glad to change my implementation.